package.json

```diff
@@ -5,6 +5,9 @@
   "author": "Douglas Christopher Wilson <doug@somethingdoug.com>",
   "license": "MIT",
   "repository": "pillarjs/finalhandler",
+  "type": "commonjs",
+  "main": "index.js",
+  "types": "types/index.d.ts",
```

Comment:
@Phillip9587 ,
I checked this again and noticed that `index.d.ts` is located in root (not in `types/` folder).
How does it work? Is this line really correct?

Reply:
```suggestion
  "types": "index.d.ts",
```